test/benchgc/README.md

```diff
@@ -38,11 +53,25 @@ python -m benchgc [OPTIONS] --driver [DRIVER] --case [CASE]
 * if driver=pattern, please provide the pre-defined pattern name, such as mlp here
 * if driver is a dialect name, please provide the detail op name to start a single op test
 
+### --entry [str]
+* default : "entry"
+* the entry name of the kernel of input mlir or generated mlir
+
 ### --seed [int]
 * set the seed to generate the test data and reprodce the test
 
 ### --verbose [int]
-* set the verbose level
+* set the verbose level, default : 0
+* 0 : NO_VERBOSE
+* 1 : MODULE_VERBOSE, print the module will be executed
+* 2 : ARG_VERBOSE, + print arg information
+* 3 : COMPARE_VERBOSE, + print threshold for comparison
+* 4 : ERROR_OUTPUT_VERBOSE, + print all error data points if failed
+* 5 : OUTPUT_VERBOSE, + print all result including passed tensor
+* 6 : INPUT_VERBOSE, + print input torch tensors
+
+### --ir_printing (action=store_true)
```

Comment:
Do you think we need to control the level of IR printing, like all stages or only changed or specified pass?

Reply:
currently the python IR printing is using MLIR python's API, it does not provide any options for controlling the printing.
```
        pm = passmanager.PassManager.parse(self.pipeline)
        if ir_printing:
            pm.enable_ir_printing()
```

Reply:
Ok, then let's keep it as is.